pkg/entity/batch_job/batch_job.go

```diff
@@ -59,29 +67,53 @@ func del(id *string, cmd *cobra.Command) {
 
 func create(cmd *cobra.Command) {
 	flags := cmd.Flags()
-	batchJobFile := util.GetStringAndErr(flags, batchJobsFileFlagName)
+	batchJobFile := util.GetStringAndErr(flags, batchJobFileFlagName)
+	batchJobType := getBatchJobType(flags)
 
-	batchJobData, err := ioutil.ReadFile(batchJobFile)
+	batchJobData, err := os.ReadFile(batchJobFile)
```

Comment:
What's the diff with `ioutil`?

Reply:
Apparently `ioutil` just calls `os` nowadays, so `os` should be used instead.